keyboards/redox/keymaps/media-CH/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2021 Shiftux <shiftux@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Use I2C or Serial, not both */
+#define USE_SERIAL
+// #define USE_I2C
+
+/* Select hand configuration */
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+#define EE_HANDS
+
+#define SPLIT_USB_DETECT
+#define SPLIT_USB_TIMEOUT 2000
```

Comment:
Is this required? 

Eg, this tends to only be needed for the Elite C v3 (currently on v4)

Reply:
i am unsure. but i removed it, compiled and flashed and evt is still working as expected (typing these lines right now). 
so i guess it is not really required. :)

Reply:
Then ideally, it should be removed. 

This setting is mostly for split keyboards that don't have proper vbus detection.  That includes the Proton C's, as well as (only) the rev3 of the Elite C.  Otherwise, it introduces a delay.

Reply:
already removed. see latest commit